README.md

```diff
@@ -112,13 +112,15 @@ These is the evolving list of TDD requirements which led to the implementation.
 * [x] Money distribution has a precision member, use that instead of the contrived Precision on Rounding.
 * [x] Add unary and binary minus and the division operator.
 * [x] The context has a smallest distribution unit.
+* [x] We can calculate a dimensionless factor by dividing two money objects.
```

Comment:
Why not

> A dimensionless factor can be calculated by dividing two money objects.

then the style of language is more consistent

Reply:
Fixed in 3b568e2a6d3bcfdcaf22911fccf6e5333e2bd004